pyaedt/modeler/modeler3d.py

```diff
@@ -452,8 +458,8 @@ def replace_3dcomponent(
         variables = []
         if variables_to_include:
             dependent_variables = []
-            ind_variables = self._app._variable_manager.independent_variable_names
-            dep_variables = self._app._variable_manager.dependent_variable_names
+            ind_variables = [i for i in self._app._variable_manager.independent_variable_names]
+            dep_variables = [i for i in self._app._variable_manager.dependent_variable_names]
```

Comment:
Out of curiosity, why do we need to use comprehension lists here.
Does it fix a bug?

Reply:
@MaxJPRey We modified it because we tested a case with hundreds of variables and it took a lot of time this loop, doing the comprehension, the loop was much faster.

Reply:
OK, makes sense. Thanks.